packages/components/src/highlight-cards/mobile-highlight-cards.tsx

```diff
@@ -0,0 +1,63 @@
+import { Icon } from '@wordpress/icons';
+import ShortenedNumber from '../number-formatters';
+import { TrendComparison } from './count-comparison-card';
+import './style.scss';
+
+type MobileHighlightCardProps = {
+	heading: string;
+	count: number | null;
+	previousCount?: number | null;
+	icon?: any;
+};
+
+function MobileHighlightCard( { heading, count, previousCount, icon }: MobileHighlightCardProps ) {
+	// We require at minimum a heading and a count.
+	const isValidHighlight = count !== null || heading.length > 0;
+	if ( ! isValidHighlight ) {
+		return null;
+	}
+	// We require two counts to display a trendline.
+	// The icon is optional.
+	const displayTrendline = count !== null && previousCount !== null;
+	const displayIcon = icon !== undefined;
+	return (
+		<div className="mobile-highlight-cards__item">
+			{ displayIcon && (
+				<span className="mobile-highlight-cards__item-icon">
+					<Icon icon={ icon } />
+				</span>
+			) }
+			<span className="mobile-highlight-cards__item-heading">{ heading }</span>
+			{ displayTrendline && (
+				<span className="mobile-highlight-cards__item-trend">
+					<TrendComparison count={ count ?? null } previousCount={ previousCount ?? null } />
```

Comment:
`??` is call the nullish coalescing operator, so I suspect whether the use of it is redundant here?

Reply:
So it's possible for `previousCount` to be undefined here, in which case the second `??` prevents that from going through. Looking at the code for the trend, it appears to handle `undefined` values though, so you're right that it isn't strictly necessary.

My intention here is to not display any trend unless we have two valid numbers though. In that case, I think the correct approach is to remove the `??` checks and update the `displayTrendline` assignment.

Thanks for the catch!